Lucidum/README.md

```diff
@@ -30,11 +30,26 @@ _sourceCategory = Lucidum
 ```
 ## Collect Logs for Lucidum
 
+### Collector URL
+https://endpoint4.collection.sumologic.com/receiver/v1/http/ZaVn************8abx9m9EAuUHJQ==
+
+### cURL Examples
+The cURL examples below point to the SumoLogic endpoint set up on 12/17/2021 to start developing Lucidum support for SumoLogic.
+
+### POST example
+```
+curl -v -X POST -T /path/to/your/logfile.log https://endpoint4.collection.sumologic.com/receiver/v1/http/ZaVn************8abx9m9EAuUHJQ==
+```
+### POST gzip compressed data
+```
+curl -v -X POST -T /path/to/your/logfile.log -H 'Content-Encoding:gzip' https://endpoint4.collection.sumologic.com/receiver/v1/http/ZaVn************8abx9m9EAuUHJQ==
+```
+
 ## Install the Lucidum App and View the Dashboards
```

Comment:
Install the app should follow this format
for ex https://github.com/SumoLogic/sumologic-public-partner-apps/tree/master/Tessian#install-the-sumo-logic-app

Whom to contact for support and some use cases